py/tables/fcoll_zreion.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/env python
```

Comment:
I think it would be good to make this an executable file that takes in the arguments of zeta, Mmin so that we can easily run it from the command line without changing anything inside the function